%\title{LaTeX Portrait Poster Template}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% a0poster Portrait Poster
% LaTeX Template
% Version 1.0 (22/06/13)
%
% The a0poster class was created by:
% Gerlinde Kettl and Matthias Weiser (tex@kettl.de)
% 
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%   PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[a0,portrait]{a0poster}

\usepackage{multicol} % This is so we can have multiple columns of text side-by-side
\columnsep=100pt % This is the amount of white space between the columns in the poster
\columnseprule=3pt % This is the thickness of the black line between the columns in the poster

\usepackage[svgnames]{xcolor} % Specify colors by their 'svgnames', for a full list of all colors available see here: http://www.latextemplates.com/svgnames-colors

\usepackage{times} % Use the times font
%\usepackage{palatino} % Uncomment to use the Palatino font

\usepackage{graphicx} % Required for including images
\graphicspath{{figures/}} % Location of the graphics files
\usepackage{booktabs} % Top and bottom rules for table
\usepackage[font=small,labelfont=bf]{caption} % Required for specifying captions to tables and figures
\usepackage{amsfonts, amsmath, amsthm, amssymb} % For math fonts, symbols and environments
\usepackage{wrapfig} % Allows wrapping text around tables and figures

%% A few new commands
\newcommand*{\barbar}[1]{\overline{\overline{#1}}}
%%%%%%%%%%%% smaller \overline (NB asymmetric emph. to right)
\newcommand{\overbar}[1]{\mkern 3mu\overline{\mkern-3mu#1\mkern-1.mu}\mkern 1.mu}

\begin{document}

%----------------------------------------------------------------------------------------
%   POSTER HEADER 
%----------------------------------------------------------------------------------------

% The header is divided into two boxes:
% The first is 75% wide and houses the title, subtitle, names, university/organization and contact information
% The second is 25% wide and houses a logo for your university/organization or a photo of you
% The widths of these boxes can be easily edited to accommodate your content as you see fit

\begin{minipage}[b]{0.75\linewidth}
\VeryHuge \color{NavyBlue} \textbf{Testing for differentially expressed genetic pathways with single-subject N-of-1 data in the presence of inter-gene correlation} \color{Black}\\ % Title
%% \Huge\textit{Country Update}\\[2.4cm] % Subtitle
[2.4cm]
\Huge \textbf{\underline{A.~Grant Schissler},~Walter W.~Piegorsch,~and Yves A.~Lussier}\\[0.5cm] % Author(s)
\huge Department of Mathematics \& Statistics, University of Nevada, Reno, NV, USA\\[0.4cm] % University/organization
{\huge Interdisciplinary Program in Statistics, Center for Biomedical Informatics and Biostatistics (CB2), BIO5 Institute, Department of Medicine, Department of Mathematics, University of Arizona, Tucson, AZ, USA}\\[0.4cm]
\Large \texttt{aschissler@unr.edu}\\
\end{minipage}
%
\begin{minipage}[b]{0.25\linewidth}
  \centering
  \includegraphics[width=8cm]{nevada-vertical-blue.png}\\
  \includegraphics[width=8cm]{uofa_logo.jpeg}\\
  \includegraphics[width=8cm]{stat_logo_REDBLU.jpg}\\

\end{minipage}

\vspace{1cm} % A bit of extra whitespace between the header and poster content

%----------------------------------------------------------------------------------------

\begin{multicols}{3} % This is how many columns your poster will be broken into, a portrait poster is generally split into 2 columns

%----------------------------------------------------------------------------------------
%   ABSTRACT
%----------------------------------------------------------------------------------------

\color{Navy} % Navy color for the abstract

\begin{abstract}
Modern precision medicine increasingly relies on molecular data analytics, wherein development of interpretable single-subject (``N-of-1'') signals is a challenging goal.  A previously developed global framework, N-of-1-\emph{pathways}, employs single-subject gene expression data to identify differentially expressed gene set pathways (DEPs) in an individual patient.  Unfortunately, the limited amount of data within the single-subject, N-of-1 setting makes construction of suitable statistical inferences for identifying DEPs difficult, especially when non-trivial inter-gene correlation is present.  We propose a method that exploits external information on gene expression correlations to cluster positively co-expressed genes within pathways, then assesses differential expression across the clusters within a pathway.
%\sout{The clustering can persuasively uncover the genetic patterns of association within pathways.}
A simulation study illustrates that the cluster-based approach exhibits satisfactory false-positive error control and reasonable power to detect DEPs.  An example with a single N-of-1 patient's triple negative breast cancer data illustrates use of the methodology.
\end{abstract}
%----------------------------------------------------------------------------------------
%   INTRODUCTION
%----------------------------------------------------------------------------------------

\color{Black} % SaddleBrown color for the introduction
\section*{Introduction}

\subsection*{Motivating data \& background}
%% \subsection{Motivating data}

Example paired RNA-seq quantified mRNA (normalized) expression data (from The Cancer Genome Atlas\cite{weinstein2013cancer} in a Gene Ontology\cite{Ashburner2000} pathway.

\begin{table}
\label{tab:TNBCdata}
\begin{tabular}{l|ccc}
\hline
Gene & Case expression & Baseline expression & Difference\\
\hline
  \emph{CYP4A11}   & 0.00  & 3.71  & -3.71 \\
  \emph{AGTR1}     & 6.13  & 7.86  & -1.73 \\
  \emph{OR51E2}    & 2.90  & 1.54  & 1.36 \\
  \emph{CYP11B2}   & 0.00  & 0.00  & \phantom{-}0.00 \\
  \emph{PTPRO}     & 3.72  & 6.22  & -2.50 \\
  \emph{CYP4F2}    & 0.00  & 0.40  & -0.40 \\
  \emph{AGT}       & 8.40  & 7.89  & \phantom{-}0.52 \\
  \emph{$\ldots$}       & $\ldots$  & $\ldots$  & $\ldots$ \\
  \emph{SERPINF2}  & 6.38  & 9.57  & -3.19 \\
\hline
\end{tabular}
\end{table}

%----------------------------------------------------------------------------------------
%   N-of-1-pathways
%----------------------------------------------------------------------------------------

\color{Black} % DarkSlateGray color for the rest of the content
\subsection*{The N-of-1-{\itshape pathways} framework}

The first attempt at a testing procedure employed a Wilcoxon signed-rank test\cite{Gardeux2014}.

\begin{center}\vspace{1cm}
\includegraphics[width=0.8\linewidth]{N-of-1-pathways-dep-flowchart}
\captionof{figure}{\color{Green} Workflow to enable precision medicioen from a par af osamples.Translat.}
\end{center}%\vspace{1cm}

% ----------------------------------------------------------------------------------------
%   Clustered-T
%----------------------------------------------------------------------------------------
\section*{Hypothesis testing:~the Clustered-$T$ statistic}
\subsection*{Motivation \& development}

Major issue:~inter-gene correlations complicate testing

We'd like to test:\\
$H_{0}$:~Pathway is not differentially expressed. \\
$H_{a}$:~Pathway is differentially expressed.\\
But mRNAs are likely co-expressed (correlated) within pathways.\\
Accounting for inter-gene correlation in gene set testing is an active area of research\cite{Wu2012,Tamayo2016}

%% \begin{center}\vspace{1cm}
%% \includegraphics[width=1.0\linewidth]{cT_r_hist_fig_S1_small}
%% \captionof{figure}{\color{Green} Pairs of genes are correlated (co-expression) within patients. Here are the perason corrrelatino coefficinecnts for all p30 chosse 2 paris of genes in the pathw,a etimeatie across 111 paitenst in tcga genome atles.}
%% \end{center}\vspace{1cm}
%% 

\subsection*{A cluster-correlated variance estimator}

  Williams, Rick L. "A note on robust variance estimation for cluster-correlated data." Biometrics 56.2 (2000): 645-646.

  \begin{tabular}{r | c | l }
    Concept & Symbol & Definition \\
    \hline \hline
      Observation index & $k$ & $k = 1,2,\ldots,n_j$ \\
      Cluster index & $j$ & $j = 1,2,\ldots,m$ \\
      Gene-wise difference & $d_{jk}$ & $c_{jk} - b_{jk}$ \\
      Total number of genes & $G$ & $G = \sum_{j} n_{j}$ \\
      Grand sum & $d_{\scriptscriptstyle{++}}$ & $\sum_j \sum_k d_{jk}$ \\
      Grand mean & $\barbar{d}$ & $d_{\scriptscriptstyle{++}}/G$ \\
      Cluster sum & $d_{j\scriptscriptstyle{+}}$ & $\sum_k d_{jk}$ \\
      Cluster mean & $\overbar{d}$ & $\sum_j d_{j\scriptscriptstyle{+}}/m$ \\
      Sample variance & $S_{d}^{2}$ & $\frac{1}{m-1}\sum_j(d_{j\scriptscriptstyle{+}} - \overbar{d})^2$\\
\end{tabular}

%------------------------------------------------

\subsection*{Williams' robust variance estimator\cite{Williams2000}}

Assume centered, cluster-correlated data:

\begin{itemize}
  
\item{$E[d_{jk}]=0$}.
  
\item{$cov[d_{jk},d_{jk^{\prime}}]=\sigma_{jkk^{\prime}}$}.
  
\item{$cov[d_{jk},d_{j^{\prime}k^{\prime}}]=0$ when $j\neq j^{\prime}$}.
  
\end{itemize}

Then an unbiased variance estimator\footnotemark~for the grand sum is
\begin{equation*}
\label{eq:var}
\widehat{\text{Var}}[d_{\scriptscriptstyle{++}}] = \frac{m}{m-1}\sum_{j=1}^m(d_{j+} - \overbar{d})^2.
\end{equation*}
%\pause
\centering Clearly, $\widehat{\text{Var}}[d_{\scriptscriptstyle{++}}] = mS_d^2$.

%------------------------------------------------

Hypotheses
  Denote $\mu = E\left(\barbar{D}\right)$ and let this quantity represent the true magnitude of pathway differential expression.\\
  \vskip6pt
  {Then the statistical hypotheses of interest are}
    \begin{equation*}
      \label{eq:hypotheses}
      \begin{array}{rl}
        H_{0}: & \mu = 0 \\
        H_{a}: & \mu \neq 0 \ .
      \end{array}
    \end{equation*}

%------------------------------------------------

Reference distribution
  
      Modeling assumptions under $H_0$
        \begin{itemize}
        \item{Conditional on the cluster assignments.}
        \item{Further assume $D_{j\scriptscriptstyle{+}} \sim N(0, \sigma^2)$.}
        \end{itemize}
%\pause
      Then the Clustered-$T$ statistic{\footnotemark}
\begin{equation*}
  \label{eq:tstat}
T =  \frac{\overbar{d}}{S_d/\sqrt{m}}
\end{equation*}
follows $T \sim t(m-1)$ under $H_0$. \\
A (two-sided) P-value is $2\times\text{Pr}[t(m-1) \ge |T|]$.

%\pause
{\footnotetext{Inferences using $T$ apply to $\barbar{D}$ and the MD score as both are constant scalings of $\overbar{D}$.}}

% ----------------------------------------------------------------------------------------
%   Monte Carol Evaluation
%----------------------------------------------------------------------------------------
\section*{Monte Carlo evaluation}
\subsection*{Simulation settings}

  \begin{center}
\begin{tabular}{lll}
Design \\variable & Description & Values\\
\hline
$G$ & Number of genes in pathway & \{15, 30, 50, 100, 200, 400\}\\
$p$ & the proportion of DEGs\footnotemark & \{0, 0.3, 0.6, 0.9\}\\
$\psi$ & fold change of DEGs & \{1.5, 2, 4\}\\
$\mathbf{R}$ & pathway correlation structure & \{Independent, Block, All\}\\
\end{tabular}
\end{center}
%\pause
\begin{itemize}
\item \lq Non-DEG \rq: \(X_{i} \sim NegBin(\hat{\mu_{i}},\hat{\delta_{i}})\)
  %\pause
\item \lq DEG \rq: \(X_{i} \sim NegBin(\psi \times \hat{\mu_{i}},\hat{\delta_{i}})\)
\end{itemize}

\footnotetext{DEG = differentially expressed gene}

% ------------------------------------------------
\subsection*{Defining realistic gene sets}
\begin{center}\vspace{1cm}
\includegraphics[width=0.8\linewidth]{cT_r_hist_fig_S1_all}
\captionof{figure}{\color{Green} Main Algerian geothermal areas (Fekraoui and Abouriche, 1995)}
\end{center}\vspace{1cm}

% ------------------------------------------------

\subsection*{Simulating pathways via copulas}
2000 simulated bivariate gene counts with specified correlation of 0.49, simulated using copulas\cite{Genest2007,Yan2007}
\begin{center}\vspace{1cm}
\includegraphics[width=0.8\linewidth]{copula_viz}
\captionof{figure}{\color{Green} Main Algerian geothermal areas (Fekraoui and Abouriche, 1995)}
\end{center}\vspace{1cm}

%------------------------------------------------

\subsection{Evaluation:~performance}
\begin{center}\vspace{1cm}
\includegraphics[width=0.8\linewidth]{clustered_T_fpr_small}
\captionof{figure}{\color{Green} Simulation results:~false positive rates}
\end{center}\vspace{1cm}

\begin{center}\vspace{1cm}
\includegraphics[width=0.8\linewidth]{clustered_T_power_small}
\captionof{figure}{\color{Green} Simulation results:~power for the Clustered-\itshape{T}}
\end{center}\vspace{1cm}

%----------------------------------------------------------------------------------------
%   Application
%----------------------------------------------------------------------------------------
\subsection{Application:single-subject inference for TNBC patient}
Four top-hit differentially expressed pathways for a TNBC patient when testing 3411 GO-BP pathways.

%% \begin{table}
%%    \label{tab:topHits}
%% %%\begin{tabular}{>{\raggedright}p{0.375\linewidth} | c | c | c |c |c}
%%   \hspace*{0pt}\makebox[\linewidth][c]{\begin{tabular}{>{\raggedright}p{0.375\linewidth} | c | c | c |c |c}
%%   Gene set description & $\barbar{D}$ & $T$-stat & P-value & $G$ & $m$ \\
%%    \hline \hline
%%   pos reg of cell adhesion & -0.75 & -4.92 & 0.00011 & 226 & 19 \\
%%   \hline
%%   reg of resp to external stimulus & -0.47 & -4.42 & 0.00015 & 458 & 28 \\
%%   \hline
%%   mitochondrial translational initiation & \phantom{-}0.28 & \phantom{-}7.55 & 0.00028 & 84 & 7 \\
%%   \hline
%%   regulation of cell morphogenesis involved in differentiation & -0.51 & -4.80 & 0.00028 & 168 & 15 \\
%%   \hline
%%   \end{tabular}
%%  }
%%  \end{table}
%% %\captionof{80 DEPs at BH 5\%, 601 DEPs at nominal 5\%}

%----------------------------------------------------------------------------------------
%   CONCLUSIONS
%----------------------------------------------------------------------------------------

\color{SaddleBrown} % SaddleBrown color for the conclusions to make them stand out


\section*{Illustrative summary}
\begin{center}\vspace{1cm}
\includegraphics[width=0.8\linewidth]{NxT_MD_fig1_summary}
\captionof{figure}{\color{Green} Simulation results:~power for the Clustered-\itshape{T}}
\end{center}\vspace{1cm}

\section*{Conclusions}
Despite being a petroleum- and gas-rich country, Algeria is making efforts to exploit its renewable energies. The Algerian government has adopted new renewable energy laws and financial support for the investors to facilitate the exploitation of the renewable energies for electricity production and direct utilizations. Algeria has relatively abundant geothermal resources especially in the northeastern parts but not totally used.
\color{Black} % Set the color back to DarkSlateGray for the rest of the content

%----------------------------------------------------------------------------------------
%   FORTHCOMING RESEARCH
%----------------------------------------------------------------------------------------

\section*{Forthcoming Research}

Simulation of thermodynamic properties of the thermal fluid and power output with longevity using geological, hydrogeological, and geothermal data from NE-Algerian geothermal reservoirs. 

%----------------------------------------------------------------------------------------
%   FORTHCOMING RESEARCH
%----------------------------------------------------------------------------------------

\section*{acknowledgements}

The results published here are in whole or part based upon data generated by the TCGA Research Network: http:// cancergenome.nih.gov/. We gratefully acknowledge the kind and helpful input of Mr. Qike Li, Dr. Joanne Berghout, Dr. Ikbel Achour, Dr. Colleen Kenost, Dr. Haiquan Li, Dr. Nima Pouladi, Dr. Ryan Gutenkunst, and Dr. Joseph Watkins. In addition, thanks are due the Editor and two anonymous referees for insightful comments that greatly improved the quality of the manuscript. This work represents a portion of the first author’s Ph.D. dissertation from the University of Arizona Graduate Interdisciplinary Program in Statistics. This material is based upon work supported by the U.S. National Science Foundation under Grant No. 1228509 and by the U.S. National Institutes of Health under Grant No. R03ES027394.
 %----------------------------------------------------------------------------------------
%   REFERENCES
%----------------------------------------------------------------------------------------

%% \nocite{*} % Print all references regardless of whether they were cited in the poster or not
\bibliographystyle{plain} % Plain referencing style
\bibliography{poster_RSS_2018_bib} % Use the example bibliography file sample.bib

%----------------------------------------------------------------------------------------

\end{multicols}
\end{document}