%\title{LaTeX Portrait Poster Template}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% a0poster Portrait Poster
% LaTeX Template
% Version 1.0 (22/06/13)
%
% The a0poster class was created by:
% Gerlinde Kettl and Matthias Weiser (tex@kettl.de)
% 
% This template has been downloaded from:
% http://www.LaTeXTemplates.com
%
% License:
% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%----------------------------------------------------------------------------------------
%   PACKAGES AND OTHER DOCUMENT CONFIGURATIONS
%----------------------------------------------------------------------------------------

\documentclass[a0,portrait]{a0poster}

\usepackage{multicol} % This is so we can have multiple columns of text side-by-side
\columnsep=100pt % This is the amount of white space between the columns in the poster
\columnseprule=3pt % This is the thickness of the black line between the columns in the poster

\usepackage[svgnames]{xcolor} % Specify colors by their 'svgnames', for a full list of all colors available see here: http://www.latextemplates.com/svgnames-colors

\usepackage{times} % Use the times font
%\usepackage{palatino} % Uncomment to use the Palatino font

\usepackage{array} % Required for tables
\usepackage{graphicx} % Required for including images
\graphicspath{{figures/}} % Location of the graphics files
\usepackage{booktabs} % Top and bottom rules for table
\usepackage[font=small,labelfont=bf]{caption} % Required for specifying captions to tables and figures
\usepackage{amsfonts, amsmath, amsthm, amssymb} % For math fonts, symbols and environments
\usepackage{wrapfig} % Allows wrapping text around tables and figures
%% \usepackage{natbib} % Allows for more flexible references

%% A few new commands
\newcommand*{\barbar}[1]{\overline{\overline{#1}}}
%%%%%%%%%%%% smaller \overline (NB asymmetric emph. to right)
\newcommand{\overbar}[1]{\mkern 3mu\overline{\mkern-3mu#1\mkern-1.mu}\mkern 1.mu}

\begin{document}

%----------------------------------------------------------------------------------------
%   POSTER HEADER 
%----------------------------------------------------------------------------------------

% The header is divided into two boxes:
% The first is 75% wide and houses the title, subtitle, names, university/organization and contact information
% The second is 25% wide and houses a logo for your university/organization or a photo of you
% The widths of these boxes can be easily edited to accommodate your content as you see fit

\begin{minipage}[b]{0.8\linewidth}
\VeryHuge \color{NavyBlue} \textbf{Testing for differentially expressed genetic pathways with single-subject N-of-1 data in the presence of inter-gene correlation~\cite{Schissler2018}} \color{Black}\\ % Title
%% \Huge\textit{Country Update}\\[2.4cm] % Subtitle
[1.0cm]
\Huge \textbf{\underline{A.~Grant Schissler$^{1,4}$},~Walter W.~Piegorsch$^{2-5}$,~and Yves A.~Lussier$^{4-6}$}\\[0.5cm] % Author(s)
\huge $^{1}$Department of Mathematics \& Statistics, University of Nevada, Reno, NV, USA\\[0.4cm] % University/organization
{\huge $^{2}$Interdisciplinary Program in Statistics, $^{3}$Department of Mathematics, $^{4}$Center for Biomedical Informatics and Biostatistics (CB2), $^{5}$BIO5 Institute, $^{6}$Department of Medicine, University of Arizona, Tucson, AZ, USA}.
%\Large \texttt{aschissler@unr.edu}\\
\end{minipage}
%
\begin{minipage}[b]{0.2\linewidth}
  \centering
  \includegraphics[width=8cm]{nevada-vertical-blue.png}\\
  \includegraphics[width=8cm]{uofa_logo.jpeg}\\
  \includegraphics[width=8cm]{stat_logo_REDBLU.jpg}\\
\end{minipage}

\vspace{0cm} % A bit of extra whitespace between the header and poster content

%----------------------------------------------------------------------------------------

\begin{multicols}{3} % This is how many columns your poster will be broken into, a portrait poster is generally split into 2 columns

%----------------------------------------------------------------------------------------
%% %   ABSTRACT
%% %----------------------------------------------------------------------------------------
 
%% \color{Navy} % Navy color for the abstract
%% 
%% \begin{abstract}
%% Modern precision medicine increasingly relies on molecular data analytics, wherein development of interpretable single-subject (``N-of-1'') signals is a challenging goal.  A previously developed global framework, N-of-1-\emph{pathways}, employs single-subject gene expression data to identify differentially expressed gene set pathways (DEPs) in an individual patient.  Unfortunately, the limited amount of data within the single-subject, N-of-1 setting makes construction of suitable statistical inferences for identifying DEPs difficult, especially when non-trivial inter-gene correlation is present.  We propose a method that exploits external information on gene expression correlations to cluster positively co-expressed genes within pathways, then assesses differential expression across the clusters within a pathway.
%% %\sout{The clustering can persuasively uncover the genetic patterns of association within pathways.}
%% A simulation study illustrates that the cluster-based approach exhibits satisfactory false-positive error control and reasonable power to detect DEPs.  An example with a single N-of-1 patient's triple negative breast cancer data illustrates use of the methodology.
%% \end{abstract}
  
% ----------------------------------------------------------------------------------------
%   INTRODUCTION
%----------------------------------------------------------------------------------------

\color{Black} % SaddleBrown color for the introduction
\section{Introduction}

We seek to assess a patient's differential RNA expression, but \textit{not} for individual genes, for collections of genes (a gene set or \textit{pathway}). We call this detecting Differentially Expressed Pathways (DEPs) as opposed to Differentially Expressed Genes (DEGs). 

\subsection{Motivating data \& background}
%% \subsection{Motivating data}

The table below contains paired RNA-seq ($log_{2}$-normalized) expression data derived from a triple negative breast cancer patient~\cite{weinstein2013cancer}. The genes listed here are contained within a Gene Ontology~\cite{Ashburner2000} pathway.\\

\begin{tabular}{l|ccc}
\hline
Gene & Tumor expression &  Normal expression & Difference\\
\hline
  \emph{CYP4A11}   & 0.00  & 3.71  & -3.71 \\
  \emph{AGTR1}     & 6.13  & 7.86  & -1.73 \\
  \emph{OR51E2}    & 2.90  & 1.54  & 1.36 \\
  \emph{CYP11B2}   & 0.00  & 0.00  & \phantom{-}0.00 \\
  \emph{PTPRO}     & 3.72  & 6.22  & -2.50 \\
  \emph{CYP4F2}    & 0.00  & 0.40  & -0.40 \\
  \emph{AGT}       & 8.40  & 7.89  & \phantom{-}0.52 \\
  \emph{$\ldots$}       & $\ldots$  & $\ldots$  & $\ldots$ \\
  \emph{SERPINF2}  & 6.38  & 9.57  & -3.19 \\
\hline
\end{tabular}

%----------------------------------------------------------------------------------------
%   N-of-1-pathways
%----------------------------------------------------------------------------------------

\color{Black} % DarkSlateGray color for the rest of the content
\subsection{The N-of-1-{\itshape pathways} clinical framework~\cite{Gardeux2014}}

\begin{center}\vspace{1cm}
\includegraphics[width=1\linewidth]{N-of-1-pathways-dep-flowchart}
\captionof{figure}{\color{Black} Conceptual workflow to enable precision medicine from within-patient paired expression data.}
\end{center}%\vspace{1cm}

% ----------------------------------------------------------------------------------------
%   Clustered-T
%----------------------------------------------------------------------------------------
\section{The Clustered-$T$ test statistic}
%%\subsection{Motivation \& rationale}
We seek to improve upon the first N-of-1-\textit{pathways} testing procedure, a Wilcoxon signed-rank test~\cite{Gardeux2014}. The major issue is that inter-gene correlations invalid modeling assumptions~\cite{Tamayo2016}.

\begin{center}\vspace{1cm}
\includegraphics[width=0.8\linewidth]{cT_r_hist_fig_S1_all}
\captionof{figure}{\color{Black} Evidence of gene co-expression within pathways:~Pearson correlation estimates across all pairs of genes within 6 Gene Ontology pathways, derived from 111 breast cancer patients.}
\end{center}\vspace{1cm}

% ------------------------------------------------
\subsection{Clustering of positively co-expressed genes}
Accounting for inter-gene correlatinot is difficutlt with only limtied samples. Instead of estimating covariance directly, we use a robust cluster-correlated variance estimator~\cite{Williams2000}. This requires that we cluster genes \textit{a priori}, using a database of samples. We omit the clustering procedure here, see Reference~\cite{Schissler2018} for details.

\subsection{Notation}
Once clustered, we develop our statistic using the following notation:\\

\begin{tabular}{r | c | l }
    Concept & Symbol & Definition \\
    \hline \hline
      Observation index & $k$ & $k = 1,2,\ldots,n_j$ \\
      Cluster index & $j$ & $j = 1,2,\ldots,m$ \\
      Gene-wise difference & $d_{jk}$ & $c_{jk} - b_{jk}$ \\
      Total number of genes & $G$ & $G = \sum_{j} n_{j}$ \\
      Grand sum & $d_{\scriptscriptstyle{++}}$ & $\sum_j \sum_k d_{jk}$ \\
      Grand mean & $\barbar{d}$ & $d_{\scriptscriptstyle{++}}/G$ \\
      Cluster sum & $d_{j\scriptscriptstyle{+}}$ & $\sum_k d_{jk}$ \\
      Cluster mean & $\overbar{d}$ & $\sum_j d_{j\scriptscriptstyle{+}}/m$ \\
    Sample variance & $S_{d}^{2}$ & $\frac{1}{m-1}\sum_j(d_{j\scriptscriptstyle{+}} - \overbar{d})^2$\\
  \end{tabular}

%------------------------------------------------
\subsection{Williams' robust variance estimator~\cite{Williams2000}}
Modeling the differences as centered and cluster-correlated ( $E[d_{jk}]=0$, $cov[d_{jk},d_{jk^{\prime}}]=\sigma_{jkk^{\prime}}$, and $cov[d_{jk},d_{j^{\prime}k^{\prime}}]=0$ when $j\neq j^{\prime}$),
%% \begin{itemize}
%% \item{$E[d_{jk}]=0$}.
%% \item{$cov[d_{jk},d_{jk^{\prime}}]=\sigma_{jkk^{\prime}}$}.
%% \item{$cov[d_{jk},d_{j^{\prime}k^{\prime}}]=0$ when $j\neq j^{\prime}$}.
%% \end{itemize}
we use an unbiased variance estimator for the grand sum:
\begin{equation}
\label{eq:var}
\widehat{\text{Var}}[d_{\scriptscriptstyle{++}}] = \frac{m}{m-1}\sum_{j=1}^m(d_{j+} - \overbar{d})^2.
\end{equation}
%\pause
Clearly, $\widehat{\text{Var}}[d_{\scriptscriptstyle{++}}] = mS_d^2$.
%------------------------------------------------

\subsection{Hypotheses \& reference distribution}
Denote $\mu = E\left(\barbar{D}\right)$. Then the statistical hypotheses of interest are
\begin{equation}
 \label{eq:hypotheses}
 \begin{array}{rl}
 H_{0}: & \mu = 0 \\
 H_{a}: & \mu \neq 0 \ .
 \end{array}
 \end{equation}
%------------------------------------------------

\noindent To build a reference distribution, model the cluster sums as $D_{j\scriptscriptstyle{+}} \sim N(0, \sigma^2)$. Then, conditional on the cluster assignments and under $H_0$, our Clustered-$T$ statistic\\
\begin{equation}
  \label{eq:tstat}
T =  \frac{\overbar{d}}{S_d/\sqrt{m}}
\end{equation}
follows a $t(m-1)$ distribution. \\
A (two-sided) P-value is $2\times\text{Pr}[t(m-1) \ge |T|]$.

%\pause
%{\footnotetext{Inferences using $T$ apply to $\barbar{D}$ and the MD score as both are constant scalings of $\overbar{D}$.}}

% ----------------------------------------------------------------------------------------
%   Monte Carol Evaluation
%----------------------------------------------------------------------------------------
\section{Monte Carlo evaluation}
We evaluate our methodology and compare to the leading alternatives, a Wilcoxon signed-rank test and an unadjusted (na\"ive) $t$-test.
\subsection{Simulation settings}
\begin{center}
\begin{tabular}{lll}
Variable & Description & Values\\
\hline
$G$ & Number of genes in pathway & \{15, 30, 50, 100, 200, 400\}\\
$p$ & the proportion of DEGs & \{0, 0.3, 0.6, 0.9\}\\
$\psi$ & fold change of DEGs & \{1.5, 2, 4\}\\
$\mathbf{R}$ & pathway correlation structure & \{Independent, Block, All\}\\
\end{tabular}
\end{center}
%\pause
\begin{itemize}
\item \lq Non-DEG \rq: \(X_{i} \sim NegBin(\hat{\mu_{i}},\hat{\delta_{i}})\)
  %\pause
\item \lq DEG \rq: \(X_{i} \sim NegBin(\color{Red}\psi \color{Black} \times \hat{\mu_{i}},\hat{\delta_{i}})\)
\end{itemize}

%\footnotetext{Block structure indicates clusters are true}
%\footnotetext{DEG = differentially expressed gene}

% ------------------------------------------------
%% \subsection{Defining realistic gene sets}

% ------------------------------------------------

\subsection{Simulating pathways via copulas}
To induce correlation, we use copulas~\cite{Yan2007}.
\begin{center}\vspace{1cm}
\includegraphics[width=0.5\linewidth]{copula_viz}
\captionof{figure}{\color{Black} 2000 simulated bivariate gene counts with specified correlation of 0.49 and heterogeneous negative binomial marginals.}
\end{center}\vspace{1cm}

%------------------------------------------------

\subsection{Evaluation:~operating characteristics}
\begin{center}\vspace{1cm}
\includegraphics[width=0.8\linewidth]{clustered_T_fpr_small}
\captionof{figure}{\color{Black} The Clustered-$T$ statistic maintains a 5\% size of the test under various correlation structures --- \lq Independent\rq~simulates genes independently, \lq Block\rq~simulates under the clustering assumptions, \lq All\rq~allows all genes to be co-expressed.}
\end{center}\vspace{1cm}

\begin{center}\vspace{1cm}
\includegraphics[width=1.0\linewidth]{clustered_T_power_small}
\captionof{figure}{\color{Black} The Clustered-$T$ displays adequate power while increasing fold change of DEGs (dash-dot = 1.5, dashed = 2, solid line = 4).}
\end{center}\vspace{0.2cm}

%----------------------------------------------------------------------------------------
%   Application
%----------------------------------------------------------------------------------------
\section{Application}
We present our patient's four top-hit differentially expressed pathways when testing 3411 GO-BP pathways. These pathways represent potential therapeutic targets to enable precision medicine.

%% \begin{table}
%%    \label{tab:topHits}
%% %%\begin{tabular}{>{\raggedright}p{0.375\linewidth} | c | c | c |c |c}
%%  \hspace*{0pt}\makebox[\linewidth][c]
\begin{tabular} {>{\raggedright}p{0.375\linewidth} | c c c c c}
   Gene set description & $\barbar{D}$ & $T$-stat & P-value & $G$ & $m$ \\
    \hline \hline
   pos reg of cell adhesion & -0.75 & -4.92 & 0.00011 & 226 & 19 \\
   \hline
   reg of resp to external stimulus & -0.47 & -4.42 & 0.00015 & 458 & 28 \\
   \hline
   mitochondrial translational initiation & \phantom{-}0.28 & \phantom{-}7.55 & 0.00028 & 84 & 7 \\
   \hline
   regulation of cell morphogenesis involved in differentiation & -0.51 & -4.80 & 0.00028 & 168 & 15 \\
   \hline
   \end{tabular}
%%  \end{table}
%% %\captionof{80 DEPs at BH 5\%, 601 DEPs at nominal 5\%}

%----------------------------------------------------------------------------------------
%   CONCLUSIONS
%----------------------------------------------------------------------------------------

\color{Black} % SaddleBrown color for the conclusions to make them stand out

\section{Concluding remarks}

\begin{center}\vspace{1cm}
\includegraphics[width=0.6\linewidth]{NxT_MD_fig1_summary}
\captionof{figure}{\color{Black} Illustrative summary. The axes represent baseline (horizontal) and tumor (vertical) expression within a pathway. The diagonal line visualizes equal expression. The coloring of each gene indicates cluster assignment. The vertical green line displays gene-wise differential expression. We use a clustered-correlated variance estimator to assess differential pathway expression.}
\end{center}\vspace{1cm}

%----------------------------------------------------------------------------------------
%   FORTHCOMING RESEARCH
%----------------------------------------------------------------------------------------

%% \subsection*{Forthcoming Research}
%% 
%% Simulation of thermodynamic properties of the thermal fluid and power output with longevity using geological, hydrogeological, and geothermal data from NE-Algerian geothermal reservoirs. 

%----------------------------------------------------------------------------------------
%   FORTHCOMING RESEARCH
%----------------------------------------------------------------------------------------

\subsection*{Acknowledgments}

This material is based upon work supported by the U.S. National Science Foundation under Grant No. 1228509 and by the U.S. National Institutes of Health under Grant No. R03ES027394.
 %----------------------------------------------------------------------------------------
%   REFERENCES
%----------------------------------------------------------------------------------------

%% \nocite{*} % Print all references regardless of whether they were cited in the poster or not
\bibliographystyle{cj} % Plain referencing style
\bibliography{poster_RSS_2018_bib} % Use the example bibliography file sample.bib

%----------------------------------------------------------------------------------------

\end{multicols}
\end{document}